lms/static/js/program_marketing.js

```diff
@@ -4,3 +4,17 @@ function playVideo(src) {
     document.querySelector('#program_video iframe').style = 'display:block;';
     document.querySelector('#program_video iframe').src = src;
 }
+function toggleBio(currentEl, bioHash) {
+    'use strict';
+    var button = currentEl;
+    var bioId = 'instructor-bio-' + bioHash;
+    var bio = document.getElementById(bioId);
+    var hidden = bio.classList.contains('hidden');
+    if (hidden) {
+        bio.classList.remove('hidden');
+        button.innerHTML = 'Hide Bio';
```

Comment:
should this text be coming out of a message catalog?

Reply:
What do you mean by message catalog?